websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,170 @@
+				state: "Premieres",
+			},
+			"/calendars/my/new-shows": {
+				details: "Browsing my calendar",
+				state: "New shows",
+			},
+			"/calendars/my/movies": {
+				details: "Browsing my calendar",
+				state: "Movies",
+			},
+			"/calendars/my/dvd": {
+				details: "Browsing my calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/calendars/shows": {
+				details: "Browsing calendar",
+				state: "Shows",
+			},
+			"/calendars/premieres": {
+				details: "Browsing calendar",
+				state: "Premieres",
+			},
+			"/calendars/new-shows": {
+				details: "Browsing calendar",
+				state: "New shows",
+			},
+			"/calendars/movies": {
+				details: "Browsing calendar",
+				state: "Movies",
+			},
+			"/calendars/dvd": {
+				details: "Browsing calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/discover": { details: "Finding new shows/movies" },
+			"/apps": { details: "Browsing apps" },
+			"/vip": { details: "Browsing VIP features" },
+			"/settings": { details: "Browsing settings" },
+			"/auth/signin": { details: "Registering on Trakt" },
+			"/auth/login": { details: "Logging in to Trakt" },
+			"/widgets": { details: "Browsing widgets" },
+		};
+
+	for (const [path, data] of Object.entries(pages))
+		if (pathname.includes(path)) presenceData = { ...presenceData, ...data };
+
+	if (pathname.includes("/users")) {
+		const username = document.querySelector(
+			"#avatar-wrapper >  .emojis-supported > a"
+		).textContent;
+		presenceData.buttons = [{ label: "Check profile", url: document.location.href }];
+		if (pathname.includes("/history"))
+			presenceData.details = `Browsing ${username}'s history`;
+		else if (pathname.includes("/progress"))
+			presenceData.details = `Browsing ${username}'s progress`;
+		else if (pathname.includes("/collection"))
+			presenceData.details = `Browsing ${username}'s collection`;
+		else if (pathname.includes("/ratings"))
+			presenceData.details = `Browsing ${username}'s ratings`;
+		else if (pathname.includes("/lists"))
+			presenceData.details = `Browsing ${username}'s lists`;
+		else if (pathname.includes("/comments"))
+			presenceData.details = `Browsing ${username}'s comments`;
+		else if (pathname.includes("/network") || pathname.includes("/network/friends"))
+			presenceData.details = `Browsing ${username}'s friends`;
+		else presenceData.details = `Browsing ${username}'s profile`;
+	} else if (pathname.includes("/shows/")) {
+		if (pathname.includes("/episodes/")) {
+			presenceData.details = "Browsing episode";
+			presenceData.state = `${
+				document
+					.querySelector(
+						"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h2 > a:nth-child(1)"
+					)
+					.textContent.split(":")[0]
+			} -
+		${
+			document.querySelector(
+				"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1 > span.main-title-sxe"
+			).textContent
```

Comment:
This displays something like SxEE where S is the season number and EE is the episode number. Can't you display the episode name? Or better yet, display the season and episode numbers along with the title?

Reply:
something like this?
![image](https://user-images.githubusercontent.com/63014799/174448448-b5095992-6aa9-4862-835d-d7c7a8b6212f.png)

Reply:
TBH you can remove the first line and just have the name of the series in there and then the name of the episode and number in the bottom line

Reply:
lgtm